kernel/univ.ml

```diff
@@ -38,20 +38,22 @@ struct
   open Names
 
   module UGlobal = struct
-    type t = DirPath.t * int
+    type t = DirPath.t * string * int
```

Comment:
May I advocate we use a integer here instead of a string? Allocation and comparison make strings expensive, and we have a *lot* of universe operations going on on a typical run.

Reply:
On 64-bit architectures it might even be completely overblown, the highest bits could be reserved for the Coq process id without having any observable effect for the end user.

Reply:
Noted, but I guess we first fix all the issues and then we scrape the bottom of the barrel